src/main/resources/config/liquibase/changelog/20231126172632_changelog.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20231126172632" author="zbil">
+        <addColumn tableName="conversation_participant">
+            <column name="is_muted" type="boolean" defaultValueBoolean="false">
+                <constraints nullable="false"/>
```

Comment:
Okay, so since this is NOT NULL and the Java type for this is `boolean`, then I would strongly encourage you to replace all instances where `Boolean` is used for this type (and possibly for the other types) with `boolean`. Otherwise you could run into NullPointerExceptions.

Due to the scope of this change you could do this in a follow-up PR.

Reply:
After looking at the code more it seems like the other booleans in the ConversationParticipant Entity are Nullable at least in Java. This should certainly be fixed in a future PR together with all the other places where `Boolean` is used for these. It just doesn't make sense here to allow null values.

And for your new attribute: Since you use `boolean` in the Java entity, you must not use `Boolean` anywhere without an additional null check, as you run the risk of NullPointerExceptions otherwise.

Reply:
I would prefer to convert `is_favorite` and `is_hidden` in another PR, since this one is open for too long.

Ref.: https://docs.liquibase.com/change-types/add-not-null-constraint.html

Reply:
Totally fine :D

Reply:
https://github.com/ls1intum/Artemis/issues/7996